run.rb

```diff
@@ -64,9 +64,9 @@
       if is_raw
         event.send "```#{JSON.pretty_generate(q)}```"
       else
-        pos = q.change_percent.positive?
-        emoji = pos ? '📈' : '📉'
-        event.send "**#{q.symbol}** is #{pos ? '' : 'not '}stonks #{emoji} `$#{q.latest_price} (#{q.change_percent_s})`"
+        msg = "**#{q.symbol}** #{bot.stonk_data(q)}"
+        msg += " after nap: #{bot.stonk_data(q, extended: true)}" if q.extended_price_time > (q.iex_last_updated || 0)
```

Comment:
```suggestion
        msg += " after nap: #{bot.stonk_data(q, extended: true)}" if (q.extended_price_time || 0) > (q.iex_last_updated || 0)
```

was throwing a no method error for `>` since the ext price time was nil

Reply:
oops I ||0'd the wrong side..nice

Reply:
classic